pattern-library/aquent.html

```diff
@@ -224,19 +224,53 @@ <h2 class="separated-white-opaque-thin-narrow">Separated with thin white underli
 
             </section>
             <hr>
-            <section id="feature-3">
-              <h3>Example Feature 3</h3>
-              <p>Galaxies the ash of stellar alchemy! Two ghostly white figures in coveralls and helmets are soflty dancing how far away Hypatia? Brain is the seed of intelligence radio telescope the carbon in our apple pies globular star cluster! Trillion
-                hearts of the stars? Cosmic fugue laws of physics, permanence of the stars cosmos gathered by gravity, not a sunrise but a galaxyrise radio telescope tingling of the spine! Cosmos billions upon billions hearts of the stars take root and
-                flourish the sky calls to us at the edge of forever cosmic ocean encyclopaedia galactica?</p>
-              <button class="button" data-toggle="unique-id">Show code</button>
-              <pre id="unique-id" data-toggler=".hide" class="hide">
-                <code>
+            <section id="buttons">
+              <h3>Buttons</h3>
+              <p>To render button styles, a "button" class is required, even on 'button' elements.</p>
+              <button type="button" name="button">unstyled button element</button><br><br>
+              <button class="button" type="button" name="button">.button (.button.primary)</button><br>
+              <button class="button secondary" type="button" name="button">.button.secondary</button><br>
```

Comment:
We have three main button colors defined:
* https://github.com/aquent/dotcms.assets/blob/master/src/aquent.com/scss/components/foundation/_settings.scss#L300

Reply:
Sweet, thank you.